download-basket/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>fi.nls.oskari</groupId>
+    <artifactId>oskari-parent</artifactId>
+    <version>1.43.0-SNAPSHOT</version>
+  </parent>
+  <groupId>fi.nls.oskari</groupId>
+  <artifactId>download-basket</artifactId>
+  <version>1.43.0-SNAPSHOT</version>
+  <name>download-basket</name>
+  <url>http://maven.apache.org</url>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <javax.mail.version>1.4.7</javax.mail.version>
+    <commons-email.version>1.1</commons-email.version>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
```

Comment:
Is there a reason why this specific version is chosen? Why not use the maintained dependency?

Reply:
fixed